### ISSUE TITLE
Clippy is your mate

### ISSUE THREAD

**Issue by omid** [pouriya/sssh#15]
Clippy is your mate, even if you sleep with it `nightly`!

**Comment by pouriya** [comment]
I'm aware of it. thanks. Currently on Linux it only generates [one warning](https://github.com/pouriya/sssh/actions/runs/4331108926/jobs/7562743210#step:8:160). Do you have any suggestion?

**Comment by omid** [comment]
yep, nightly will return a few more. Some might be false negative.
For example:
`cargo +nightly clippy --fix --allow-dirty --allow-staged --all-features --all-targets -- -W clippy::all -W clippy::pedantic -W clippy::cargo`
